content/en/news/releases/1.6.x/announcing-1.6/change-notes/index.md

```diff
@@ -0,0 +1,61 @@
+- ***Added*** the ability for mesh operators to add and remove labels used in Istio metrics, based on expressions over the set of available request and response attributes. This improves Istio's support for [customizing v2 metrics generation](/docs/tasks/observability/metrics/customize-metrics/).
+- ***Updated*** default telemetry v2 configuration to avoid using host header to extract destination service name at the gateway. This prevents unbound cardinality due to an untrusted host header, and implies that destination service labels are going to be omitted for requests that hit `blackhole` and `passthrough` at the gateway.
+- ***Added*** automated publishing of Grafana dashboards to `grafana.com` as part of the Istio release process. Please see the [Istio org page](https://grafana.com/orgs/istio) for more information.
+- ***Updated*** Grafana dashboards to adapt to the new Istiod deployment model.
+
+## Configuration Management
+
+## Installation
+
+- ***Added*** support for Istio in-place upgrades. See the [Upgrade guide](/docs/setup/upgrade/) for more information.
+- ***Removed*** the legacy Helm charts.
+- ***Added*** the ability for users to add a custom hostname for istiod.
+- ***Changed*** gateway readiness port used from 15020 to 15021. If you check health on your Istio `ingressgateway` from your Kubernetes network load balancer you will need to update the port.
+- ***Added*** functionality to save installation state in a `CustomResource` in the cluster.
+- ***Changed*** the Istio installation to no longer manage the installation namespace, allowing more flexibility.
+- ***Removed*** the Citadel, Sidecar Injector, and Galley deployments. These were disabled by default in 1.5, and all functionality has moved into Istiod.
+- ***Removed*** the legacy `istio-pilot` configurations, such as Service.
+- ***Removed*** ports 15029-15032 from the default `ingressgateway`. It is recommended to expose telemetry addons by [host routing](/docs/tasks/observability/gateways/) instead.
+- ***Removed*** built in Istio configurations from the installation, including the Gateway, `VirtualServices`, and mTLS settings.
+- ***Added*** a new profile, called `preview`, allowing users to try out new experimental features that include WASM enabled telemetry v2.
+- ***Added*** `istioctl install` command as a replacement for `istioctl manifest apply`.
+
+## Operator
+
+## istioctl
+
+- ***Added*** better display characteristics for the istioctl command.
+- ***Added*** support for key:value list selection when using --set flag paths.
+- ***Added*** support for deletes and setting non-scalar values when using the Kubernetes overlays patching mechanism.
+
+## Miscellaneous changes
+
+- ***Added*** new and improved Istio documentation. For more information, see [Website content changes](/about/log/).
```

Comment:
Shouldn't this be taken as a given? It's an odd thing to see in release notes

Reply:
This line item is mainly useful to advertise the little-known page that actually tells users which doc pages have been updated for very major and minor release.